client/components/cards/attachments.jade

```diff
@@ -3,6 +3,13 @@ template(name="cardAttachmentsPopup")
     li
       input.js-attach-file.hide(type="file" name="file" multiple)
       a.js-computer-upload {{_ 'computer'}}
+    li
+      a.js-upload-clipboard-image {{_ 'clipboard'}}
```

Comment:
You should add the corresponding key in the `i18n/en.i18n.json` file (sorted by alphabetical order).